src/main/java/frc/robot/commands/ClimberGoTo.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.Constants;
+
+public class ClimberGoTo extends CommandBase {
```

Comment:
All commands should have a JavaDoc description of what they do so that people unfamiliar with the code can easily tell what it does without needing to decypher the entire file.

This applies to ALL of the commands, not just this one.